internal/e2e/e2ehooks/e2ehooks_test.go

```diff
@@ -0,0 +1,186 @@
+package e2ehooks
+
+import (
+	"context"
+	"errors"
+	"net/http/httptest"
+	"strings"
+	"testing"
+	"testing/iotest"
+	"time"
+
+	"github.com/supabase/auth/internal/conf"
+	"github.com/supabase/auth/internal/hooks/v0hooks"
+)
+
+func TestInstance(t *testing.T) {
+	{
+		globalCfg, err := conf.LoadGlobal("../../../hack/test.env")
+		if err != nil {
+			t.Fatalf("exp nil err; got %v", err)
+		}
+		globalCfg.DB.Driver = ""
+		globalCfg.DB.URL = "invalid"
+
+		inst, err := New(globalCfg)
+		if err == nil {
+			t.Fatal("exp non-nil err")
+		}
+		if inst != nil {
+			t.Fatal("exp nil *Instance")
+		}
+	}
+
+	{
+		globalCfg, err := conf.LoadGlobal("../../../hack/test.env")
+		if err != nil {
+			t.Fatalf("exp nil err; got %v", err)
+		}
+
+		inst, err := New(globalCfg)
+		if err != nil {
+			t.Fatalf("exp nil err; got %v", err)
+		}
+		if inst == nil {
+			t.Fatal("exp non-nil *Instance")
+		}
+		if err := inst.Close(); err != nil {
+			t.Fatalf("exp nil err from Close; got %v", err)
+		}
+	}
+}
```

Comment:
This test also doesn't add that much value. If `New` failed I'm pretty sure we wouldn't be able to release anything 😅 

The expectations also test non-nil, no error... pretty expected.

Reply:
The value is that It takes test coverage from `96.8%` to `100%`

Reply:
Yea I don't think it's worth adding 30 LOC to cover 10 in this case

Reply:
I disagree here, can we have a call to work through this?